op-e2e/actions/interop/dsl/dsl.go

```diff
@@ -374,3 +375,84 @@ func (d *InteropDSL) AdvanceL2ToLastBlockOfOrigin(chain *Chain, l1OriginHeight u
 		d.AddL2Block(chain)
 	}
 }
+
+func (d *InteropDSL) ActSyncSupernode(t helpers.Testing, opts ...actSyncSafeOption) {
+	cfg := &actSyncSafeConfig{
+		ChainOpts: d.defaultChainOpts(),
+	}
+	for _, opt := range opts {
+		opt(cfg)
+	}
+
+	preSeqSyncStatuses := make([]*eth.SyncStatus, 0, len(cfg.Chains))
+	preSupSyncStatuses := make([]*eth.SupervisorChainSyncStatus, 0, len(cfg.Chains))
+	for _, chain := range cfg.Chains {
+		preSeqSyncStatus, preSupSyncStatus := d.Actors.SyncStatuses(t, chain)
+		preSeqSyncStatuses = append(preSeqSyncStatuses, preSeqSyncStatus)
+		preSupSyncStatuses = append(preSupSyncStatuses, preSupSyncStatus)
+	}
+
+	// Perform actions
+	if cfg.shouldSendL1LatestSignal {
+		d.Actors.Supervisor.SignalLatestL1(t)
+	}
+	if cfg.shouldSendL1FinalizedSignal {
+		d.Actors.Supervisor.SignalFinalizedL1(t)
+	}
+	for _, chain := range cfg.Chains {
+		// Move supervisor safe head forward by one block
+		chain.Sequencer.SyncSupervisor(t) // supervisor to react to exhaust-L1
+	}
+	d.Actors.Supervisor.ProcessFull(t)
+	for _, chain := range cfg.Chains {
+		chain.Sequencer.ActL2PipelineFull(t) // node to complete syncing to L1 head.
+	}
+
+	for i, chain := range cfg.Chains {
+		preSeqSyncStatus := preSeqSyncStatuses[i]
+		preSupSyncStatus := preSupSyncStatuses[i]
+
+		postSeqSyncStatus, postSupSyncStatus := d.Actors.SyncStatuses(t, chain)
+
+		// Post assertions
+		require.Equal(t, preSeqSyncStatus.UnsafeL2.ID(), postSeqSyncStatus.UnsafeL2.ID(), "sequencer unsafe head should not move for chain id %s", chain.ChainID)
+
+		// TODO: Add more postSupSyncStatus assertions
+		if cfg.requireLocalSafeAdvances {
+			require.Equal(t, preSeqSyncStatus.LocalSafeL2.Number+1, postSeqSyncStatus.LocalSafeL2.Number, "sequencer safe head should move forward by one block for chain id %s", chain.ChainID)
+			// No supervisor sync status localSafe field
+		} else {
+			require.Equal(t, preSeqSyncStatus.LocalSafeL2.ID(), postSeqSyncStatus.LocalSafeL2.ID(), "sequencer local safe head should not move for chain id %s", chain.ChainID)
+		}
+
+		if cfg.requireCrossUnsafeAdvances {
+			require.Equal(t, preSeqSyncStatus.CrossUnsafeL2.Number+1, postSeqSyncStatus.CrossUnsafeL2.Number, "sequencer cross unsafe head should move forward by one block for chain id %s", chain.ChainID)
+		} else {
+			require.Equal(t, preSeqSyncStatus.CrossUnsafeL2.ID(), postSeqSyncStatus.CrossUnsafeL2.ID(), "sequencer cross unsafe head should not move for chain id %s", chain.ChainID)
+		}
+
+		if cfg.requireSafeAdvances {
+			require.Equal(t, preSeqSyncStatus.SafeL2.Number+1, postSeqSyncStatus.SafeL2.Number, "sequencer safe head should move forward by one block for chain id %s", chain.ChainID)
+			if !cfg.isPreInterop {
+				require.Equal(t, preSupSyncStatus.LocalUnsafe.ID(), postSupSyncStatus.CrossSafe, "supervisor safe head should move forward by one block for chain id %s", chain.ChainID)
```

Comment:
According to the msg string, should this check be
```suggestion
				require.Equal(t, preSupSyncStatus.CrossSafe.Number + 1, postSupSyncStatus.CrossSafe.Number, "supervisor safe head should move forward by one block for chain id %s", chain.ChainID)
```
?

Reply:
This logic has been removed in favor of {Superchain,Chain}SyncStatusAsserter.